cfgov/unprocessed/js/routes/common.js

```diff
@@ -21,7 +21,6 @@ require( '../modules/form-validation' ).init();
 require( '../modules/reveal-on-focus' ).init();
 require( '../modules/scroll-on-history-collapse' ).init();
 require( '../modules/clear-form-buttons' ).init();
-require( '../modules/jquery/custom-input' ).init();
```

Comment:
We prob shouldn't remove this till the existing forms have been updated. The blog is getting the incorrect styles atm.